storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2676,6 +2676,14 @@ private void assertTopoActive(String topoName, boolean expectActive) throws NotA
         return tryReadTopoConf(topoId, topoCache);
     }
 
+    private StormTopology tryReadTopologyFromName(final String topoName) throws NotAliveException,
+            AuthorizationException, IOException {
+        IStormClusterState state = stormClusterState;
```

Comment:
This seems redundant

Reply:
Indeed. But looks like this idiom is widely used in Nimbus.